README.md

```diff
@@ -229,9 +223,9 @@ Edit [`config/lms/production.py`](./config/lms/production.py) and add it to `edx
 
 Edit [`config/cms/production.py`](./config/cms/production.py) and add it to `edx-platform/cms/envs/`.
 
-In production, don't forget to set appropriate urls in `/opt/openedx/lms.env.json` and `/opt/openedx/cms.env.json`. Note that you will need to choose different domain names for the LMS and the CMS. In particular, pick an appropriate value for the `SITE_NAME`.
+In production, don't forget to set appropriate urls in `/opt/Open edX/lms.env.json` and `/opt/Open edX/cms.env.json`. Note that you will need to choose different domain names for the LMS and the CMS. In particular, pick an appropriate value for the `SITE_NAME`.
 
-Usually, the lms is configured on the bare domain name ("myopenedx.com") or  on the `www` subdomain ("www.myopenedx.com"), while cms typically uses the `studio` subdomain ("studio.myopenedx.com").
+Usually, the lms is configured on the bare domain name ("myOpen edX.com") or  on the `www` subdomain ("www.myOpen edX.com"), while cms typically uses the `studio` subdomain ("studio.myOpen edX.com").
```

Comment:
I think you did a global search & replace that went wrong :) The domain names should definitely not contain white space.